modules/oe_webtools_cookie_consent/tests/src/Functional/ConfigurationTest.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_webtools_cookie_consent\Functional;
+
+use Drupal\oe_webtools_cookie_consent\CookieConsentEventInterface;
+use Drupal\Tests\BrowserTestBase;
+
+/**
+ * Tests that the configured settings are correctly output in the page.
+ *
+ * @group oe_webtools_analytics
+ */
+class ConfigurationTest extends BrowserTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static $modules = [
+    'config',
+    'system',
+    'oe_webtools',
+    'oe_webtools_cookie_consent',
+  ];
+
+  /**
+   * Tests if the configuration for the Webtools library is present on the page.
+   */
+  public function testLibraryLoading(): void {
```

Comment:
Are this tests not reduntant with the Behat tests? I think instead we could test the events?

Reply:
This test checks the cache invalidation.

Reply:
:+1: